### PR TITLE
zdemo_excel29 demo file from Web Repository

### LINKED ISSUE #988: Have `resources/TestMacro.xlsm` file made available directly in `ZDEMO_EXCEL29`
As explained by [@AndreaBorgia-Abo](https://github.com/abap2xlsx/abap2xlsx/issues/908#issuecomment-991575796), the [FAQ](https://abap2xlsx.github.io/abap2xlsx/FAQ) says:
- "Macro-Enabled workbook \
  Run report [ZDEMO_EXCEL29](https://github.com/abap2xlsx/abap2xlsx/blob/master/src/demos/zdemo_excel29.prog.abap) and use as VBA source file [TestMacro.xlsm](https://github.com/abap2xlsx/abap2xlsx/blob/master/resources/TestMacro.xlsm) attached. Basically abap2xlsx works using an existing VBA binary (we do not want to create a VBA editor)."

Instead, I think that it would be better to provide the file in the Web Repository (`SMW0`) and add an option in `ZDEMO_EXCEL29`, to run it with that file, much like the program [`zdemo_excel_fill_template`](https://github.com/abap2xlsx/abap2xlsx/blob/master/src/demos/zdemo_excel_fill_template.prog.abap) with [`zdemo_excel_template`](https://github.com/abap2xlsx/abap2xlsx/blob/master/src/demos/zdemo_excel_template.w3mi.data.xlsx) from the Web Repository:
image

### PR DESCRIPTION
**Pull request by sandraros**
Fix #988

### CODE REVIEW

**Comment by sandraros** [comment]
@AndreaBorgia-Abo Could you make this PR a priority so that we can move the useless big folders to a new repository (#908)? Thanks!

File: src/demos/zdemo_excel29.prog.xml [hunk 1]
```diff
@@ -13,15 +13,33 @@
    <TPOOL>
     <item>
      <ID>R</ID>
-     <ENTRY>abap2xlsx Demo: Marco-Enabled workbook</ENTRY>
```
Comment (AndreaBorgia-Abo) [inline src/demos/zdemo_excel29.prog.xml:16]: Some guy will be very sad about this ;)

**Review by AndreaBorgia-Abo — APPROVED** [review]
LGTM

File: src/demos/zdemo_excel29.prog.xml (hunk 1 above)
Comment (sandraros) [reply src/demos/zdemo_excel29.prog.xml:16]: Didn't see that one :) At least, the screen title is fixed and this one doesn't show up if I'm not wrong.

Comment (AndreaBorgia-Abo) [reply src/demos/zdemo_excel29.prog.xml:16]: You fixed it all right, that was the joke :)

**Pull request merged**